src/PrestaShopBundle/Resources/config/services/bundle/repository.yml

```diff
@@ -138,6 +138,11 @@ services:
     alias: PrestaShopBundle\Entity\Repository\AttributeGroupRepository
     deprecated: ~
 
+  PrestaShopBundle\Entity\Repository\AttributeRepository:
```

Comment:
If you move the method you added in this repo in the other one I believe you can clean this definition